src/components/About/Team/index.js

```diff
@@ -133,18 +134,61 @@ const Description = styled.div`
   padding: 1rem;
 `;
 
+const contributorsLink =
+  'https://api.github.com/repos/fossasia/susi.ai/contributors';
+
```

Comment:
use api from all susi.ai repositories and add those contributors as well.

Reply:
Done bro